src/resolvers/ResolverGenerator.ts

```diff
@@ -0,0 +1,68 @@
+import { readFileSync } from 'fs';
+import { GlobSync } from 'glob';
+import * as handlebars from 'handlebars';
+import { basename, join } from 'path';
+import { ResolverInstance } from '../resolvers/ResolverInstance';
+import { HandlebarsHelpers } from '../schema/Helpers';
+
+/**
+ * Generate Typescript compatible schema
+ */
+export class GraphQLResolverGenerator {
+  private rootTemplate: string = `{{> resolvers}}`
+
+  /**
+   * Register helper functions for Handlebars
+   */
+  constructor() {
+    new HandlebarsHelpers().registerHelpers()
+  }
+
+  /**
+   * Generate Typescript compatible schema
+   * @param resolvers Array of ResolverInstance
+   */
+  public generateResolvers(resolvers: ResolverInstance[]): string {
+    this.registerPartials()
+
+    const template = handlebars.compile(this.rootTemplate)
+    
+    const input: any = {}
+    input.resolvers = this.splitByTypes(resolvers)
+
+    return template(input)
+  }
+
+  /**
+   * Split resolvers into Query and Mutations
+   * @param resolvers Array of ResolverInstance
+   */
+  private splitByTypes(resolvers: ResolverInstance[]) {
```

Comment:
As mentioned above - this code makes no sense. 
Why not actually generate those objects so we do not need to split data by types - we will have everything ready.

Reply:
Hmm, seems better than this.